### PR TITLE
updating module versions

### PR DESCRIPTION
**Pull request by rryke**
## What 
Updating underlying module versions to make use of some new features. Particularly port mapping in the container definition. 

Also added in specific container ports for the ingress 

## Why 
I needed the  capability to run containers on nonstandard ports via fargate

### CODE REVIEW

File: outputs.tf [hunk 1]
```diff
@@ -17,3 +17,8 @@ output "task_role_arn" {
   description = "ECS Task role ARN"
   value       = "${module.ecs_alb_service_task.task_role_arn}"
 }
+
+output "ecs_sg_id" {
```
Comment (osterman) [inline outputs.tf]: Let's keep the output consistent. Rename to `service_security_group_id`

Comment (rryke) [reply outputs.tf]: ahh yeah, from before...

**Comment by osterman** [comment]
@rryke `terraform fmt`

**Comment by rryke** [comment]
I can't tell what its cranky about

**Comment by osterman** [comment]
image

**Pull request merged**